content/releases/cloudnative-pg-1-25.0-released/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "CloudNativePG 1.25.0 and 1.24.2 Released!"
+date: 2024-12-23
+draft: false
+author: gbartolini
+image:
+    url: 58364dde1adc4a0a8.37186404-2048x1445.jpg
+    attribution: from <a href="https://wordpress.org/photos/photo/58364dde1a/">Saurabh</a>
+tags:
+ - release
+ - postgresql
+ - postgres
+ - kubernetes
+ - k8s
+ - cloudnativepg
+ - preview
+ - cnpg
+summary: The CloudNativePG community is thrilled to unveil the release of version 1.25.0, packed with exciting new features, along with the maintenance update 1.24.2 for the 1.24.x series.
+---
+
+The **CloudNativePG Community** is delighted to announce the release of
+**CloudNativePG Operator** versions **1.25.0** and **1.24.2**!
+
+Additionally, we have released **1.23.6** as the final patch for the
+now-unsupported 1.23.x series to ensure a smooth transition for remaining
+users.
```

Comment:
"remaining users" looks a bit negative IMO, like people are leaving the platform.
How about

```suggestion
now-unsupported 1.23.x series to ensure a smooth transition for users
still using 1.23 or older releases.
```